### DISCUSSION TITLE
VR 50 Ambisense

### DISCUSSION THREAD

**Discussion by lyonanderson** [john30/ebusd#739] [Q&A]
I have built ebusd from head and am using a ebuzzz adapter kindly built for me by a member. It's working well and I can get data from my VR66, VR720 and the boiler:

ebusctl i
version: ebusd 22.4.v22.4-28-gbc3e6110
update check: revision v22.4 available
device: /dev/ttyUSB0
signal: acquired
symbol rate: 22
max symbol rate: 168
min arbitration micros: 617
max arbitration micros: 3021
min symbol latency: 4
max symbol latency: 9
reconnects: 0
masters: 4
messages: 648
conditional: 3
poll: 164
update: 11
address 00: master #1
address 03: master #11
address 05: slave #1, scanned "MF=Vaillant;ID=VR921;SW=2602;HW=5703"
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0414;HW=7401", loaded "vaillant/bai.308523.inc" ([HW=7401]), "vaillant/08.bai.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=72000;SW=0122;HW=7703", loaded "vaillant/15.720.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 64: slave, scanned "MF=Vaillant;ID=V6500;SW=0107;HW=7602", loaded "vaillant/64.v65.csv"

I have 6 of the VR 50 Ambisense radiator valves. I believe they talk to the VR921 via RF.  It's a long shot but I was hoping they would be exposed via the ebus too.  They might not be as the VR921 manages the heating on and off based on the values it's getting from the valve, I assume. 

Has anyone else had success? Is there anything I can try to get a bit more information?

**Comment by Tomasonx** [comment]
@lyonanderson 

Do you have any news about ebusd + Ambisense?

**Comment by JonesGW-MGD** [comment]
Your interpretation is similar to mine (I've got 13 VR50s + a VR52 repeater!). 

This traffic is over 868Mhz from VR50 to VR920/921.  I think the VR920/921 uses ebus to talk with the controller -  the VR66.  And obviously out to the Vaillant cloud - and the SENSOapp.  I think the info in the first instance is far less granular than in the latter instance - every TRV setting goes to the cloud - I think only the highest set point goes to the VR66.

On my logs, ebusd sees the 920 but doesn't have any config for it:

```
[main error] unable to load scan config 05: no file from vaillant with prefix 05 matches ID "vr920", SW2801, HW5703
[main error] scan config 05: ERR: element not found
``` 

I think then what happens is that the controller (VR66) sets the 'Desired Temperature' to the highest value of the reporting TRVs on the controller - I can see this then on my VR720f SENSOComfort - which in 'ambisense mode' simply echos the desired temp, you don't use it as a roomstat anymore of course.  

There are some smarts that clearly going on when the temp approaches the set points  so a bit cleverer than the usual on/off TRVs like tado etc. but I don't think any special Load compensation is happening to modify the flow temp outside of the weather compensation I've got set up.

So maybe some probing into ebusd basv z1ActualRoomTempDesired tempv which is where Desired Temperature' pops up - at least on my set up but I doubt you're going to see each TRV just the value of the 'loudest' one screaming for heat!

**Reply by steve-joyner** [reply]
I know this is an old thread and I know ambisense is now discontined by Vaillant.

However, i'm wondering if anyone ever figured out in any detail how the room temps from the ambisense trvs or room stats are communicated to the rest if the system.  Is the VR920/921. sending them all as ebus messages on the local bus, or just the highest one of those assigned to a particular zone.  Anyone have any examples of the ebus output of a VR920 or 921 ?
